### PR TITLE
Fixes #10945 - foreman-debug to collect foreman http request log

### PR DESCRIPTION
**Pull request by pmoravec**
foreman-debug to collect /var/log/httpd/foreman-ssl_access_ssl.log*

Signed-off-by: Pavel Moravec pmoravec@redhat.com

### CODE REVIEW

File: script/foreman-debug [hunk 1]
```diff
@@ -266,6 +266,7 @@ add_files /var/lib/puppet/ssl/certs/$(hostname -f).pem /var/lib/puppet/ssl/certs
 add_files /etc/{httpd,apache2}/conf/*
 add_files /etc/{httpd,apache2}/conf.d/*
 add_files /var/log/{httpd,apache2}/*error_log*
+add_files /var/log/httpd/foreman-ssl_access_ssl.log*
```
Comment (domcleal) [inline script/foreman-debug]: and the apache2 directory

**Comment by pmoravec** [comment]
Re @domcleal : thanks, updated in recent commit

**Comment by domcleal** [comment]
[test]

**Comment by dLobatog** [comment]
Merged as f51ba790b4a32ad429fe51a62220026d3c8a0a06, thanks @pmoravec!

**Pull request closed without merging**